### ISSUE TITLE
Version 3.9.6 was not found in the local cache

### ISSUE THREAD

**Issue by teror4uks** [actions/setup-python#552]
**Description:**
Log: 
```
Run actions/setup-python@v4
  with:
    python-version: 3.9.6
    check-latest: false
    update-environment: true

Version 3.9.6 was not found in the local cache
Error: Version 3.9.6 with arch x64 not found
The list of all available versions can be found here: https://raw.githubusercontent.com/actions/python-versions/main/versions-manifest.json
```

**Action version:**
Specify the action version

**Platform:**
- [x] Ubuntu
- [ ] macOS
- [ ] Windows

**Runner type:**
- [ ] Hosted
- [ ] Self-hosted

**Tools version:**

- `3.9.6`

**Repro steps:**  

```yaml
name: Continuous Integration

on:
  push:
    branches: [main]

jobs:
  setup:
    runs-on: ubuntu-latest
    timeout-minutes: 5

    - name: Set up Python
      uses: actions/setup-python@v4
      with:
        python-version: 3.9.6

```

**Expected behavior:**
to find the python version in the latest ubuntu container

**Actual behavior:**
failed to find

**Comment by MaksimZhukov** [comment]
Hello @teror4uks !
The `ubuntu-latest` tag is being migrated to Ubuntu 22.04 (see details in the related [announcement](https://github.com/actions/runner-images/issues/6399)). We have not built Python 3.9.6 for Ubuntu 22.04. Python 3.9 is available for installation on Ubuntu 22.04 starting with Python 3.9.12. Is it possible for you to specify the Python version as `3.9.12` (or higher) or as `3.9`?
```yml
    - name: Set up Python
      uses: actions/setup-python@v4
      with:
        python-version: '3.9'
```

**Comment by teror4uks** [comment]
yeah I think it works for me, thanks, I will change it to `3.9`
